### PR TITLE
Run Sylius tests on SyliusDemo

### PR DESCRIPTION
**Pull request by Zales0123**
Based on https://github.com/Sylius/SyliusDemo/pull/26

Regarding excluded features see: https://github.com/Sylius/SyliusDemo/issues/29

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -38,6 +38,7 @@
         "coduo/php-matcher": "^2.1",
         "friends-of-behat/context-service-extension": "^1.0",
         "friends-of-behat/cross-container-extension": "^1.0",
+        "friends-of-behat/exclude-specifications-extension": "dev-fix-excluding-feature-files",
```
Comment (Zales0123) [inline composer.json]: To be changed after merging https://github.com/FriendsOfBehat/ExcludeSpecificationsExtension/pull/22

Comment (pamil) [reply composer.json]: v0.1.1 released :)

File: composer.json [hunk 2]
```diff
@@ -20,10 +20,10 @@
         }
     ],
     "require": {
-        "php": "^7.0",
+        "php": "^7.1",
 
-        "sylius/sylius": "^1.0",
-        "sylius/shop-api-plugin": "^1.0@beta",
+        "sylius/sylius": "^1.1",
```
Comment (pamil) [inline composer.json]: Can we make it `^1.1@rc` and remove `"minimum-stability": "RC"`?

**Comment by pamil** [comment]
Mateusz Zalewski, you're great!

**Comment by Zales0123** [comment]
Kamil Kokot, thank you!

**Pull request merged**